specification/amqp_0.9.1/src/main/scripts/org/kaazing/specification/amqp/tcp/open/identity/response.rpt

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+####################################################
+## AMQP server (QPID)
+####################################################
+
+# META INFO: Accepting at epoch Mon Oct 06 19:37:07 PDT 2014  -  1.4116669279825E9
+#accept tcp://127.0.0.1:8011
```

Comment:
Could you just include the contents of the tcp.accept script here? Then you can remove the tcp.accept.rpt script. Or if you really do need to keep them separate please remove these two commented out lines.